packages/shim/src/sdk.ts

```diff
@@ -197,27 +198,27 @@ export function addBreadcrumb(breadcrumb: Breadcrumb): void {
 }
 
 /**
- * Updates user context information for future events.
- * @param extra User context object to merge into current context.
- */
-export function setUserContext(user: User): void {
-  invokeClient('setContext', { user });
-}
-
-/**
- * Updates tags context information for future events.
- * @param extra Tags context object to merge into current context.
- */
-export function setTagsContext(tags: { [key: string]: string }): void {
-  invokeClient('setContext', { tags });
-}
-
-/**
- * Updates extra context information for future events.
- * @param extra Extra context object to merge into current context.
- */
-export function setExtraContext(extra: object): void {
-  invokeClient('setContext', { extra });
+ * Callback to set context information onto the scope
+ *
+ * @param scope Scope
```

Comment:
@param here is a `callback` function that will receive `scope` as an argument, not the `scope` itself